packages/wallet/backend/src/email/service.ts

```diff
@@ -49,4 +55,23 @@ export class EmailService implements IEmailService {
       `Send email is disabled. Reset password link is: ${url}`
     )
   }
+
+  async sendVerifyEmail(to: string, token: string): Promise<void> {
+    const domain = to.split('@')[1]
+    await verifyDomain(domain)
+
+    const url = `${this.baseUrl}/auth/verify/${token}`
+
+    if (this.deps.env.SEND_EMAIL) {
+      return this.send({
+        to,
+        subject: '[Rafiki.Money] Verify your account',
+        html: getVerifyEmail(url)
```

Comment:
I thought this was a function that check the email again. We should name this accordingly - that it retrieves the email template for a verification email.